modules/ROOT/pages/server-compatibility-eventing.adoc

```diff
@@ -67,9 +63,55 @@ For Sync Gateway versions that write import XATTRs:
 
 NOTE: If the `import_mutation_aware` flag is set to `true`, the performance of the Eventing function will drop because every mutation processed by Eventing will require a subdocument operation to maintain a cursor or progress for any Function that is sharing a Sync Gateway endpoint.  
 
+The procedure to enable Sync Gateway support for an Eventing function is as follows:
+
+. On Couchbase Server, xref:server:eventing:eventing-api.adoc#basic_pause[pause the function].
+
+. Using the Eventing REST API, set the `allow_sync_documents` setting for the function to false.
++
+NOTE: The `allow_sync_documents` setting is true by default.
+You must also include the `deployment_status` and `processing_status` settings in the request body.
```

Comment:
I'm having some doubts as to whether `allow_sync_documents` _is_ true by default. The PRD says it is, but the spec in the Eventing repo says it isn't. Let's just delete this line for now. I'll correct the information in the REST API documentation when I know more.

```suggestion
You must also include the `deployment_status` and `processing_status` settings in the request body.
```

Reply:
Done!